### PR TITLE
'Failing' type to 'Degraded' change in nw operator use cases

### PR DESCRIPTION
**Pull request by anuragthehatter**
Nothing big but just changed 'Failing' type to 'Degraded' in scenarios due to https://github.com/openshift/cluster-network-operator/pull/147 .Confirmed Tests works fine after this change
PTAL @bmeng @zhaozhanqi

### CODE REVIEW

**Comment by anuragthehatter** [comment]
Confirmed all test worked fine locally after latest commit (networking.rb and all nw operator scenarios). Thanks!

File: features/networking/operator.feature [hunk 1]
```diff
@@ -23,9 +23,15 @@ Feature: Operator related networking scenarios
...
     Given evaluation of `cluster_version('version').version` is stored in the :ocp_version clipboard
+<<<<<<< HEAD
     #Making sure that operator is not Degraded before proceesing further steps
     And evaluation of `cluster_operator('network').condition(type: 'Degraded')` is stored in the :Degraded_status_before_patch clipboard
     Then the expression should be true> cb.Degraded_status_before_patch["status"]=="False"
+=======
+    #Making sure that operator is not Failing before proceesing further steps
+    And evaluation of `cluster_operator('network').condition(type: 'Failing')` is stored in the :failing_status_before_patch clipboard
+    Then the expression should be true> cb.failing_status_before_patch["status"]=="False"
+>>>>>>> e96c99592f57c99857183b780d5c9f6471a84fdb
```
Comment (akostadinov) [inline features/networking/operator.feature]: garbage in commit it seems

File: features/networking/operator.feature [hunk 2]
```diff
@@ -23,9 +23,12 @@ Feature: Operator related networking scenarios
...
     #Getting OCP version
     Given evaluation of `cluster_version('version').version` is stored in the :ocp_version clipboard
-    #Making sure that operator is not Failing before proceesing further steps
-    And evaluation of `cluster_operator('network').condition(type: 'Failing')` is stored in the :failing_status_before_patch clipboard
-    Then the expression should be true> cb.failing_status_before_patch["status"]=="False"
+    #Making sure that operator is not Degraded before proceesing further steps
+    And evaluation of `cluster_operator('network').condition(type: 'Degraded')` is stored in the :Degraded_status_before_patch clipboard
+    Then the expression should be true> cb.Degraded_status_before_patch["status"]=="False"
+    #Making sure that operator is not Degraded before proceesing further steps
+    And evaluation of `cluster_operator('network').condition(type: 'Degraded')` is stored in the :Degraded_status_before_patch clipboard
```
Comment (akostadinov) [inline features/networking/operator.feature]: as a matter of style could you use lower case in clipboard names?

Comment (akostadinov) [reply features/networking/operator.feature]: other than that looks good to me

Comment (anuragthehatter) [reply features/networking/operator.feature]: Thanks for catching that @akostadinov . Fixed it. Didn't mean to make them capital but seems like :%s// operation in VI editor made it along with others.

**Pull request merged**